src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,5 +1,61 @@
 package frc.robot.subsystems;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import frc.robot.Constants.ArmConstants;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class IntakeSubsystem {
-    
+public class IntakeSubsystem extends SubsystemBase{
+  private CANSparkMax leftIntakeMotor, rightIntakeMotor, topIntakeMotor;
+  private double intakePower;
+  private double intakeOff = 0;
+  private double intakeOn = ArmConstants.intakeVoltage;
+  private final double HARD_STOP = 25.0;
```

Comment:
`HARD_STOP` isn't a very descriptive name, maybe we should call this `CURRENT_LIMIT`